### DISCUSSION TITLE
Callsign beacon meant to be one-shot?

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#294] [Help & Questions] [answered]
Using example.py from the RNode firmware, the callsign is not broadcast until a packet is sent, and then it is broadcast directly before the requested packet. It will not repeat again during the id_interval, but will broadcast again before the next outgoing packet.

My current implementation, based on the current RNodeInterface, there is no broadcast until the first packet is sent, and then it begins a cooldown where the callsign is sent at the end of the interval.

In either case, there is only a single broadcast of the callsign unless a packet is sent since the last broadcast. Which is the intended behavior? Is the single broadcast working as intended?

**Comment by kc1awv** [comment, marked as answer]
I think the idea is that transmitting the callsign in Reticulum is for identification only. Under at least the US rules, you identify at the start, end, and every 10 minutes of a 'conversation', whether it's digital or analog. id_interval might just be for setting that time requirement according to local rules/laws? If you're not actively in a conversation, there's no need to identify and keeps the frequency available for others.

If this is the case, a beacon would be a different function, but useful nonetheless. A periodic beacon would help keep the mesh alive, or at least keep other nodes aware of each other. I do run nomadnet on a local machine attached to a TNC and an RNode, so announces and such keep other local nodes aware of the wireless link.

**Reply by faragher** [reply]
Of course! That's why it's "id_callsign" not "beacon"

I keep forgetting Reticulum is meant to be run on amateur bands as well. I should get my technician's license to know these regs. I hear they got rid of the Morse requirement. Yes, I know it's been a while.
